recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - arrow-cpp >=0.12
+    - bisonpp
+    - blosc
+    - boost-cpp 1.68
+    - boost 1.68
+    - clangdev 6
+    - cmake
+    - cython
+    - double-conversion
+    - flex
+    - gdal
+    - gflags
+    - git
+    - glog
+    - go
+    - libarchive
+    - libkml
+    - libpng
+    - llvmdev 6
+    - make
+    - maven
+    - openjdk 8
+    - openssl
+    - pytest
+    - python >=3.6
+    - setuptools
+    - numba >=0.40
+    - numpy
+    - parquet-cpp
+    - pyarrow >=0.12
```

Comment:
Do you need pyarrow at build time?

Reply:
not sure. @pearu do you know that?

Reply:
pyarrow is not needed at build time, it's a python package..
pyarrow is used by pymapd

Reply:
mmm ok .. I just asked that because I am running the sanity_tests inside build time .. and I couldn't figure it out what tests is running there yet. Thanks @pearu I will remove that!